t/Makefile.am

```diff
@@ -158,6 +163,24 @@ std_stdio_gotcha_t_SOURCES = \
   std/fflush.c \
   std/size.c
 
+sys_statfs_gotcha_t_SOURCES = \
+  sys/statfs_suite.h \
+  sys/statfs_suite.c \
+  sys/statfs.c
+
+sys_statfs_gotcha_t_CPPFLAGS = $(test_cppflags)
+sys_statfs_gotcha_t_LDADD = $(test_gotcha_ldadd)
+sys_statfs_gotcha_t_LDFLAGS = $(test_gotcha_ldflags)
+
+std_stdio_gotcha_t_SOURCES = std/stdio_suite.h \
+                             std/stdio_suite.c \
+                             std/fopen-fclose.c \
+                             std/fseek-ftell.c \
+                             std/fwrite-fread.c \
+                             std/fflush.c \
+                             std/size.c
+>>>>>>> accbacf (test: add configuration to check tmpfs magic)
```

Comment:
This might be the causing the current failure of 
```
config.status: error: cannot find input file: `t/Makefile.in'
```

One other thing to check is, for some reason, my editor insisted on using tabs instead of whitespace only within this makefile, with was causing other issues for me.

Reply:
Thanks, @CamStan .  Yep, good eye.  I had a stray line from a bad merge conflict resolution.